### ISSUE TITLE
Update RNA-seq Ortholog Mapping Analysis: Use a Results Table

### ISSUE THREAD

**Issue by cbethell** [AlexsLemonade/refinebio-examples#304]
Per [@jashapiro's comment](https://github.com/AlexsLemonade/refinebio-examples/pull/267#pullrequestreview-509594046) on the PR handling the addition of the RNA-seq ortholog mapping analysis to the repo, we will likely want to update this example analysis to show how to use it in the context of some other results: i.e a DE results table for which a user may want to see what the orthologs were.

The idea here would be showing the ortholog mapping example in a different context that is also not so _excessive_ as a full gene expression matrix.

Feel free to leave any additional thoughts on updating this analysis to show users ortholog mapping in a different context from the [microarray example](https://github.com/AlexsLemonade/refinebio-examples/blob/master/02-microarray/ortholog-mapping_microarray_01_ensembl.Rmd).

**Comment by cansavvy** [comment]
Is this something we think should be done "before going live"? #286

**Comment by cbethell** [comment]
I do not believe this has to be done "before going live". What do you think @jashapiro?

**Comment by jashapiro** [comment]
I have no particular opinion... I wouldn't call it high priority.
